README.md

```diff
@@ -310,6 +310,38 @@ julia> @show y
 a = 63.0
 ```
 
+#### Calling custom MATLAB function
+
+Adding the location of the function to path, then call the function:
+
+```julia
+mat"addpath('path/to/folder')"
+val = mat"myfunction($arg1, $arg2)"
+```
+If there is a  MATLAB file like this:
+
+```matlab
+function [r,u] = test(x, y)
+	r = x + y;
+	u = x - y;
+end
+```
+
+then we can call it in Julia file:
+
+```julia
+using MATLAB
+
+x = range(-10.0, stop=10.0, length=500)
+y = range(2.0, stop=3.0, length=500)
+
+mat"addpath('path of folder')"
```

Comment:
```suggestion
mat"addpath('path/to/folder')"
```
?

Reply:
Thank your very much for your advices. This is my first time to  make a PR, I learned a lot from your suggestions. I have revised  the file.